README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
do you have git configured to convert newlines from LF to CRLF on checkout ? If yes, it is the reason.

This git setting should be avoided as altering the source code can break the code. It is much better to use an editor which supports using Unix newlines even on windows (which means any text editor except notepad.exe AFAIK, and no sane people would use notepad.exe to write code)